packages/components/src/button/README.md

```diff
@@ -62,7 +62,7 @@ Since a high-emphasis button commands the most attention, a layout should contai
 
 #### Text label
 
-All button types use text labels to describe the action that happens when a user taps a button. If there’s no text label, there should be an icon to signify what the button does.
+All button types use text labels to describe the action that happens when a user taps a button. If there’s no text label, there should be a [label](#label), and eventually an icon, to signify what the button does.
```

Comment:
I might change the phrasing to something like this: "If there’s no text label, there needs to be a [label](#label) added and an icon to signify what the button does." In my mind, there needs to be both so it's more about including adding a label along with the icon.